src/puppetlabs/puppetdb/http/factsets.clj

```diff
@@ -1,16 +1,31 @@
 (ns puppetlabs.puppetdb.http.factsets
   (:require [net.cgrand.moustache :refer [app]]
+            [puppetlabs.puppetdb.http :as http]
             [puppetlabs.puppetdb.http.facts :as facts]
             [puppetlabs.puppetdb.http.query :as http-q]
             [puppetlabs.puppetdb.middleware :refer [verify-accepts-json validate-query-params
                                                     wrap-with-paging-options wrap-with-parent-check]]
             [puppetlabs.puppetdb.query.paging :as paging]
-            [puppetlabs.puppetdb.query-eng :refer [produce-streaming-body]]))
+            [puppetlabs.puppetdb.query-eng :as eng]))
+
+(defn factset-status
```

Comment:
But an absolute query is just the universal query (select *) with some restrictions on it.